ipinfo/client.go

```diff
@@ -64,21 +65,46 @@ func NewClient(
 	}
 }
 
+// newRequest for IPV4
```

Comment:
Let's not create 2 different functions here, just call directly with `true` or `false` directly into `newRequestBase`